site/content/en/latest/user/private-key-provider.md

```diff
@@ -0,0 +1,269 @@
+---
+title: "Using Envoy Private Key Provider for TLS termination"
+---
+
+The Envoy Private Key Provider (https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/common.proto#extensions-transport-sockets-tls-v3-privatekeyprovider) allows implementing different provider which can provide the private key operation acceleration through the hardware or private key protection through the hardware.
```

Comment:
can we simplify this line, so users can better understand why they would be interested in this feature

Reply:
ok, let me try. thanks